department-of-veterans-affairs/va.gov-cms
# VACMS-18703: Updating core to 10.3.2.

## Description

Closes #_18703_. 

## Testing done
Local file generation.

## Screenshots


## QA steps

What needs to be checked to prove this works?
Pull branch locally, run ddev drush cr then ddev drush composer install.
What needs to be checked to prove it didn't break any related things?
All site should function as normal.
What variations of circumstances (users, actions, values) need to be checked?
ECA module was a problem so anything that touches that.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
